front/pages/api/v1/w/[wId]/assistant/conversations/index.ts

```diff
@@ -260,13 +269,18 @@ async function handler(
         contentFragment: newContentFragment ?? undefined,
       });
       return;
+    case "GET":
+      const conversations = await getUserConversations(auth);
+      res.status(200).json({ conversations });
+      return;
```

Comment:
Should we update the swagger doc ?

Reply:
Yes I said it in OP but running npm run docs did not change anything 😬

Reply:
ok I need to update manually the swagger doc and create the new schema for conversationwithoutcontent

Reply:
Discussed IRL we don't necessarily want to comm now on it -> need to decide how we expose the ConversationWithoutContent, not a big deal but not blocking and we want to have an MVP 😬